api/src/main/java/org/openmrs/module/locationbasedaccess/utils/LocationUtils.java

```diff
@@ -79,4 +79,13 @@ public static String getUserAccessibleLocationUuid(User authenticatedUser) {
         }
         return accessibleLocationUuid;
     }
+
+    public static PersonAttributeType getPersonAttributeTypeForLBACModule() {
```

Comment:
What do you think of making this function as "CreatePersonAttributeTypeForLocationUuid"? 
Then migrate all the relevant implementation to here. (Just need to call this method from the module activator to create !

Reply:
I guess `createPersonAttributeTypeForLocation` describe more than  `createPersonAttributeTypeForLocationUuid`.

Reply:
Go ahead